locales/it/index.yml

```diff
@@ -2252,10 +2252,11 @@ bonus:
           currentPeriodUnlockBody: "Complimenti {{name}}! Hai superato la soglia di {{transactions}} transazioni e sbloccato il Cashback!"
           currentPeriodMaxAmount: "Complimenti {{name}}! Hai raggiunto l’importo massimo previsto!"
           currentPeriodSuperCashback: "In questo momento avresti diritto al Super Cashback di {{superCashbackAmount}} €! Continua così per rimanere in classifica."
-          gracePeriodBody: "Questo periodo del Cashback si è chiuso il {{date}}. Stiamo aspettando di ricevere le ultime transazioni, per calcolare i rimborsi. Entro il {{endGracePeriodDate}} potrai vedere il calcolo definitivo."
-          closedPeriodOKBody: "Complimenti {{name}}! \nHai diritto a € {{amount}} di Cashback"
-          closedPeriodMaxAmount: "è il massimo del rimborso possibile!"
-          moneyTransfer: "Entro il {{date}} riceverai il bonifico sull’IBAN che ci hai indicato."
+          gracePeriodBody: "Questo periodo del Cashback si è chiuso il {{date}}. Stiamo aspettando di ricevere le ultime transazioni, per calcolare i rimborsi. Riceverai un messaggio quando l'operazione verrà completata."
+          closedPeriodOKBody: "Complimenti {{name}}! \nHai diritto ad un rimborso di € {{amount}}"
+          closedPeriodMaxAmount: ", il massimo del rimborso possibile"
```

Comment:
@thisisjp Could this comma be misleading?
![Schermata 2021-06-23 alle 16 37 54](https://user-images.githubusercontent.com/822471/123116795-7ffa3980-d441-11eb-968a-aef63d192614.png)

Reply:
If so, then Italians should adopt dot as decimal separator to avoid any confusion :P